python/cudf/cudf/tests/test_rolling.py

```diff
@@ -24,16 +24,16 @@ def test_rollling_series_basic(data, index, agg, nulls, center):
     if len(data) > 0:
         if nulls == "one":
             p = np.random.randint(0, len(data))
-            data[p] = None
+            data[p] = np.nan
```

Comment:
Why change these to `np.nan`?

Reply:
Them being set to `None` was making Pandas coerce the Series to an `object` dtype which was then making essentially the entire downstream test worthless.

Making it `np.nan` coerces it to a `float64` instead of `object`, which we then convert to `null` in our `Column` constructor.